src/main/scala/org/broadinstitute/dsde/firecloud/HealthChecks.scala

```diff
@@ -82,5 +62,20 @@ class StartupChecks(app: Application, registerSAs: Boolean = true)
     }
   }
 
+  def healthMonitorChecks: () => Map[Subsystem, Future[SubsystemStatus]] = () => {
+    Map(
+      Agora -> app.agoraDAO.status,
+      Consent -> app.consentDAO.status,
+      GoogleBuckets -> app.googleServicesDAO.status,
+      LibraryIndex -> app.searchDAO.status,
+      OntologyIndex -> app.ontologyDAO.status,
+      Rawls -> app.rawlsDAO.status,
+      Sam -> app.samDAO.status,
+      Thurloe -> app.thurloeDAO.status,
+      adminSaRegistered -> isAdminSARegistered.map(SubsystemStatus(_, None)),
```

Comment:
I removed the initial checks at startup because this code does it, it was already side effecty.

I would argue to leave it as is because it a) is no worse than it was before and b) is lower complexity because there is not some other async process to deal with. This is also more self healing.

Reply:
The invocation of health checks at startup is FireCloudServiceActor.scala lines 66-67.

If the registration stays in the health check, maybe turn `logger.info(s"$name is properly registered.")` down to debug? Otherwise it logs once a minute, at least when I run locally.